cmd/bosun/sched/alertRunner.go

```diff
@@ -19,16 +19,25 @@ func (s *Schedule) Run() error {
 	type alertCh struct {
 		ch     chan<- *checkContext
 		modulo int
+		shift  int // used to distribute alert runs
 	}
 	chs := []alertCh{}
+
+	// Every alert gets a small shift in time.
+	// This way the alerts with the same period are not fired
+	// simultaneously, but are distributed.
+	circular_shifts := make(map[int]int)
```

Comment:
Perhaps just a note here about what the keys and values represent in this map. I believe it is "run every value" to the next offset. Probably just comment that.

Reply:
The map is `run period` → `time shift to add`.
I will add a comment.
Thanks for the feedback!